example/12_reduce/reduce_threadwise_multi_d_impl.hpp

```diff
@@ -0,0 +1,307 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
Suggestion:
All new files copyright should be 2024 instead of 2018-2023.

Reply:
done